main/test/ca/uwaterloo/flix/tools/manifests/authors_missing.toml

```diff
@@ -0,0 +1,20 @@
+[package]
```

Comment:
Its a bit cumbersome, but it would be better if these were inlined into `TestManifestParser.scala`.

Scala has excellent support for raw strings, so you can write:

```
let manifest = 
   """
   | ...
   | ...
"""
```

(If you are using idea it helps a lot with the right indentation and formatting.

Reply:
But how do I read them as files then? Since the parse function needs a Path to read the file

Reply:
Lets make the parse function that takes a string public and test that :)